File: Array-1/sameFirstLast/SameFirstLast.java
package sameFirstLast;

public class SameFirstLast {
    //Retorne true se o "length" do Array for maior ou igual a 1 e o último elemento coincide com o primeiro.

    public boolean sameFirstLast(int[] nums) {
        return nums.length >=1 && nums[0] == nums[nums.length-1];
    }
}

File: Python/List-1/same_first_last.py
"""
Dada uma lista numérica "nums", retorne true se ela possuir uma tamanho de pelo menos
"1" e o seu primeiro elemento coincidir com o último.
"""


def same_first_last(nums):
    return len(nums) >= 1 and nums[0] == nums[-1]


print(same_first_last([1, 2, 3, 1]))
